src/pages/Organize/Uploads/Move/index.js

```diff
@@ -14,20 +14,196 @@
  You should have received a copy of the GNU General Public License along
  with this program; if not, write to the Free Software Foundation, Inc.,
  51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
-***************************************************************/
+*/
 
-import React from "react";
+import React, { useState, useEffect } from "react";
+import InputContainer from "../../../../components/Widgets/Input";
+import Button from "../../../../components/Widgets/Button";
+import Alert from "../../../../components/Widgets/Alert";
+import { moveUpload, copyUpload } from "../../../../services/organizeUploads";
+import { getUploadsFolderId } from "../../../../services/organizeUploads";
+import { getFolders } from "../../../../services/getFolder";
 
 const UploadMove = () => {
+  const initialState = {
+    folderId: 1,
+    groupName: "",
+  };
+  const initialFolderList = [
+    {
+      id: 1,
+      name: "Software Repository",
+      description: "Top Folder",
+      parent: null,
+    },
+  ];
+  const initialMessage = {
+    type: "success",
+    text: "",
+  };
+  const [moveCopyUploadData, setMoveCopyUploadData] = useState(initialState);
+  const [moveCopyUploadsList, setMoveCopyUploadsList] = useState([]);
+  const [folderId, setFolderId] = useState(1);
+  const [folderList, setFolderList] = useState(initialFolderList);
+  const [uploadList, setUploadList] = useState([]);
+  const [showMessage, setShowMessage] = useState(false);
+  const [message, setMessage] = useState(initialMessage);
+
+  const handleChange = (e) => {
+    if (e.target.multiple) {
+      return setMoveCopyUploadsList(
+        Array.from(e.target.selectedOptions, (option) => option.value)
+      );
+    }
+    setMoveCopyUploadData({
+      ...moveCopyUploadData,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const getUploadList = () => {
+    getUploadsFolderId(parseInt(folderId))
+      .then((res) => {
+        setUploadList(res);
+      })
+      .catch((error) => {
+        setMessage({
+          type: "danger",
+          text: error.message,
+        });
+        setShowMessage(true);
+      });
+  };
+  const handleMove = (e) => {
+    e.preventDefault();
+    moveCopyUploadsList.length > 0 &&
+      moveCopyUploadsList.map((id) => {
+        moveUpload(
+          moveCopyUploadData.folderId,
+          parseInt(id),
+          moveCopyUploadData.groupName
+        )
+          .then(() => {
+            setMessage({
+              type: "success",
+              text: "Successfully moved the upload",
```

Comment:
Please change the message to indicate `202` response. Same for copy.

Reply:
@GMishx updated, please review it.